test/hotspot/gtest/utilities/test_growableArray.cpp

```diff
@@ -435,6 +441,10 @@ class GrowableArrayTest : public ::testing::Test {
   }
 };
 
+// static empty vector
+const GrowableArray<int> empty_array(mtTest);
+const GrowableArrayCHeap<int, mtTest> empty_cheap_array;
```

Comment:
Where are these used?

Reply:
Thanks for your reviews, these are additional to ensure that `GrowableArray` instances can be created in static context. I have added a test to check these are truly empty.

Reply:
HotSpot denigrates non-local objects with non-trivial ctors.  We shouldn't have such instances.
https://github.com/openjdk/jdk/blame/4dd64b49716144cc697fb461ff88860e2cbcaaea/doc/hotspot-style.md#L1200-L1205
See also https://github.com/openjdk/jdk/pull/24689